README.md

```diff
@@ -122,10 +124,14 @@ and provides a deeper view into our goals for the Carbon project and language.
 
 ## Project status
 
-Carbon is currently an experimental project. We want to better understand
-whether we can build a language that meets our successor language criteria, and
-whether the resulting language can gather a critical mass of interest within the
-larger C++ industry and community.
+Carbon Language is currently an experimental project. There is no working
+compiler or toolchain. You can see the demo interpreter for Carbon on
+[compiler-explorer.com](http://carbon.compiler-explorer.com/). We are looking
+for language designers to help us finish an initial design of the language.
```

Comment:
I'd rather put the call for contributions below where we talk about what we are focused on and what we are prioritizing. Would that work? I would make a suggestion there but I can't because of GitHub...

Here would be the suggested change for this part:
```suggestion
[compiler-explorer.com](http://carbon.compiler-explorer.com/). 
```

But I'd maybe change below from:

```
We are currently focused on ... design.
```

to something along the lines of:

```
If you're interested in contributing, we would love help [completing the 0.1 language designs](...), and [completing the Carbon Explorer implementation of this design](...).  We are also currently working to get more broad feedback and participation from the C++ community.
```

Basically focusing what kind of material contributions make sense.

Reply:
I've updated the PR. Is that what you were looking for?

Reply:
Yeah, that's great. Thanks!